test/test_ssl.py

```diff
@@ -291,17 +291,13 @@ def test_cert_ssl_validation_hostname_matching(self):
         #
         #   --sslPEMKeyFile=/path/to/pymongo/test/certificates/server.pem
         #   --sslCAFile=/path/to/pymongo/test/certificates/ca.pem
-        ctx = get_ssl_context(
-            None, None, None, True, None, True, False)
+        ctx = get_ssl_context(None, None, None, None, True, True, False)
```

Comment:
Just to be sure I understand this correctly, these function calls were already correct but are included in this diff simply because you moved around the arguments in the `get_ssl_context` signature, right?

Reply:
This test was okay but a lot of the other calls in this file were incorrect.